R/task.R

```diff
@@ -387,6 +399,42 @@ task_log_value <- function(id, root = NULL) {
 }
 
 
+##' @rdname task_log
+##'
+##' @inheritParams task_wait
+##'
+##' @inheritParams logwatch::logwatch
+##'
+##' @export
+task_log_watch <- function(id, poll = 1, skip = 0, timeout = Inf,
```

Comment:
So skip is not yet supported, but will be...

Reply:
yeah, I think this will be useful when moving things around over ssh potentially